addon/initializers/ember-feature-flags.js

```diff
@@ -1,9 +1,12 @@
 import Ember from 'ember';
 import ifFeature from 'ember-feature-flags/helpers/if-feature';
+import features from '../features';
 
 export default {
   name: 'ember-feature-flags',
   initialize: function( container, application ) {
+    features.setup(Ember.merge({}, application.FEATURES));
```

Comment:
From @aexmachina: Re. "setting your features from a source other than the config", we could avoid that by only calling setup() if the features are specified in the config.